hackathon_site/dashboard/frontend/src/pages/Dashboard/Dashboard.test.tsx

```diff
@@ -32,30 +32,43 @@ const mockedGet = get as jest.MockedFunction<typeof get>;
 const hardwareUri = "/api/hardware/hardware/";
 const categoriesUri = "/api/hardware/categories/";
 const teamUri = "/api/event/teams/team/";
+const ordersUri = "/api/event/teams/team/orders/";
+
+const teamAPIResponse = { data: mockTeam } as AxiosResponse<Team>;
+
+const mockOrderAPI = (orders?: Order[]) =>
+    when(mockedGet)
+        .calledWith(ordersUri)
+        .mockResolvedValue(makeMockApiListResponse<Order>(orders ?? mockOrders));
+const mockTeamAPI = (withDelay?: boolean) =>
+    when(mockedGet)
+        .calledWith(teamUri)
+        .mockResolvedValue(
+            withDelay ? promiseResolveWithDelay(teamAPIResponse, 500) : teamAPIResponse
+        );
 
 describe("Dashboard Page", () => {
     it("Renders correctly when the dashboard appears 4 cards and 3 tables", async () => {
-        const response = { data: mockTeam } as AxiosResponse<Team>;
-        when(mockedGet).calledWith(teamUri).mockResolvedValue(response);
-
+        mockTeamAPI();
```

Comment:
What does calling the function do?

Reply:
This questions makes me think you browsed over the code without trying to understand it 🤔. Try to find where the mockTeamAPI() function is and look at the function body, you've definitely wrote the code in this function before so I think you'll know what it does. I put that code inside a function so I could reuse it in other unit tests which is a better practice than copy + pasting it over and over again.